### ISSUE TITLE
Install Error (0x80073CF6)

### ISSUE THREAD

**Issue by Jerry12798** [LSPosed/MagiskOnWSA#92]
I just forked, ran a workflow, downloaded the x64 version, and ran the install script. I can see it's failing to install because of the `Add-AppxPackage -Register .\AppxManifest.xml` command in the last line of the install script. It seems it's because the `AppxManifest.xml` file is inaccessible however I'm able to see it inside the main directory of the extracted x64 zip.

[Error Screenshot](https://user-images.githubusercontent.com/48601562/142261682-36b0da76-c9e3-404e-8db6-bdf2cb3d8563.png)
[WSA Error.txt](https://github.com/LSPosed/MagiskOnWSA/files/7557133/WSA.Error.txt)

**Comment by nahid6970** [comment]
try running as administrator privilege. i also faced it and running with administrative option fixed it

**Comment by Jerry12798** [comment]
> try running as administrator privilege. i also faced it and running with administrative option fixed it

Yea I'm running PowerShell as administrator.. it seems the package may not be registering as well as some file(s) not being accessible but I'm not sure why or what.

**Comment by herrwinfried** [comment]
> > try running as administrator privilege. i also faced it and running with administrative option fixed it
> 
> Yea I'm running PowerShell as administrator.. it seems the package may not be registering as well as some file(s) not being accessible but I'm not sure why or what.

You started Windows Powershell as administrator right? If Normal Powershell is selected, it may be because of it. 🤔

**Comment by Jerry12798** [comment]
> > > try running as administrator privilege. i also faced it and running with administrative option fixed it
> 
> > 
> 
> > Yea I'm running PowerShell as administrator.. it seems the package may not be registering as well as some file(s) not being accessible but I'm not sure why or what.
> 
> 
> 
> You started Windows Powershell as administrator right? If Normal Powershell is selected, it may be because of it. 🤔 

Yes I was running in PowerShell as Administrator

**Comment by Jerry12798** [comment]
My issue has been resolved. In case anyone faces similar issues/errors I will provide my solution. I noticed my unzipped artifact was 1.83GB when it showed as being 1.85GB on my GitHub workflow. I decided to redownload the artifact and this time when I unzipped it was the 1.85GB as GitHub shown. This is odd because my first download completed normally and didn't otherwise show any signs of being incomplete. No matter the case once I ran the install script from this artifact and WSA, Google Play, Magisk, and Amazon AppStore was all successfully installed.

**Comment by Jerry12798** [comment]
My issue has been resolved. In case anyone faces similar issues/errors I will provide my solution. I noticed my unzipped artifact was 1.83GB when it showed as being 1.85GB on my GitHub workflow. I decided to redownload the artifact and this time when I unzipped it was the 1.85GB as GitHub shown. This is odd because my first download completed normally and didn't otherwise show any signs of being incomplete. No matter the case once I ran the install script from this artifact and WSA, Google Play, Magisk, and Amazon AppStore was all successfully installed.
